Automattic/woocommerce-payments
# Add track events when BNPL is enabled/disabled

Fixes #6204 

#### Changes proposed in this Pull Request

For merchants who agreed to participate in WooCommerce analytics, update and record the following events:

 - When UPE is enabled or disabled, include `is_bnpl_affirm_afterpay_enabled` flag with the event.
 - When Payment Method is enabled or disabled, send an appropriate event with `payment_method_id` property.

#### Testing instructions

* Use a BNPL-enabled merchant account for testing
* Go to the WooCommerce Payments Settings page
* Trigger the following conditions in no particular order:
  - Enable additional payment methods 
  - Disable additional payment methods
  - Enable BNPL payment methods one by one
  - Disable BNPL payment methods one by one
 * Ensure that events appear in the Tracks to reflect performed changes

No specific unit tests are in place, since it's a complicated matter to capture a sent event. Manual testing has been done to ensure events are being sent at the correct time and have correct properties.

-------------------


**Post merge**
